src/app-layout/visual-refresh-toolbar/drawer/styles.scss

```diff
@@ -5,13 +5,27 @@
 @use '../../../internal/styles/tokens' as awsui;
 @use '../../../internal/styles' as styles;
 @use '../../constants.scss' as constants;
+@use '../../../internal/generated/custom-css-properties/index.scss' as custom-props;
+
+@mixin desktop-only {
+  @include styles.media-breakpoint-up(styles.$breakpoint-x-small) {
+    @content;
+  }
+}
+
+@mixin mobile-only {
+  @include styles.media-breakpoint-down(styles.$breakpoint-x-small) {
+    @content;
+  }
+}
 
 .drawer {
   position: sticky;
-  z-index: constants.$drawer-z-index;
+  z-index: 850;
   background-color: awsui.$color-background-container-content;
   display: grid;
   grid-template-columns: awsui.$space-m 1fr;
+  inline-size: var(#{custom-props.$drawerSize});
```

Comment:
To manage the size of the global drawers, see: https://github.com/cloudscape-design/components/blob/feat/app-layout-global-drawers/src/app-layout/visual-refresh-toolbar/drawer/global-drawer.tsx#L99

Since local drawers inherit their size from the parent element, they don't need this prop, only the global drawers uses it